src/main/java/minegame159/meteorclient/mixin/ChunkOcclusionDataBuilderMixin.java

```diff
@@ -20,5 +22,6 @@
     private void onMarkClosed(BlockPos pos, CallbackInfo info) {
         ChunkOcclusionEvent event = MeteorClient.EVENT_BUS.post(ChunkOcclusionEvent.get());
         if (event.isCancelled()) info.cancel();
+        if (Modules.get().get(NoRender.class).noCaveCulling()) {info.cancel();}
```

Comment:
Just above is an event so why didn't you use the event instead of hardcoding the module?

Reply:
I used the event in my addon which adds a No Cave Culling module, but when I went to add the functionality to NoRender I saw that most of the work for other settings is done outside of the module class.  It was just an attempt at keeping the No Render class more uniform.  I can change it back if necessary

Reply:
Yeah, other stuff in No Render are not events just because they are not needed by anything else.